src/lib/hooks/useTokenList/fetchTokenList.ts

```diff
@@ -43,31 +47,40 @@ export default async function fetchTokenList(
     urls = uriToHttp(listUrl)
   }
 
+  if (urls.length === 0) {
+    throw new Error('Unrecognized list URL protocol.')
+  }
+
+  /**
+   * Try each of the derived URLs until one succeeds.
```

Comment:
I think the `/**` should be just `/*` unless this is intended to be a JSDoc which it doesn't seem like it should be?

Reply:
also since its one line could just be //

Reply:
sounds good